pkg/config/generate.go

```diff
@@ -131,15 +132,28 @@ func Generate(m *api.Manifest, c *Config) (err error) {
 	}{
 		// Generate etcd certs
```

Comment:
Hrm, isn't the operator handling these?

Reply:
If you want TLS we need to generate it:
https://github.com/coreos/etcd-operator/blob/master/doc/user/cluster_tls.md